mcping/Cargo.toml

```diff
@@ -10,12 +10,17 @@ repository = "https://github.com/Scetch/mcping"
 edition = "2018"
 readme = "README.md"
 
+[features]
+future = ["tokio", "async-trait"]
```

Comment:
I'd prefer to see this feature named `async`, I think.

Reply:
I also wanted to name it `async` but naming a module a reserved keyword seems like a bad idea, and I'd like to keep the feature and module name the same.

Reply:
Ah that's fair, I overlooked the module name issue there. How do you feel about naming the feature `tokio-runtime` and naming the module `tokio`?